### PR TITLE
Lunnum's Return (Black Skull of Xikma Quest)

### PR DESCRIPTION
**Pull request by sLackeR-hub**
**Added**

**NPC**
Fiun Layeel

**Portals**
Bookcase
Temple Entrance

**Items**
Fiun Crown
Lunnums Chest
Lunnums Token
Lunnums Chest
Doors/Keys
Blue Face Paint Recipe

**Creatures**
Afesse Sclavus
Illu Sclavus
Lunnum
Patrician Kilkitris/Phaerix
Storage Crate

**Quests**
Lunnum's Return

**Landblocks**
00EF
9EE8
536D

### CODE REVIEW

File: Database/Patches/2006-10-DanceOfTheDead/9 WeenieDefaults/Creature/Unsorted/87150 Storage Crate.sql [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+SET @parent_id = LAST_INSERT_ID();
+
+INSERT INTO `weenie_properties_emote_action` (`emote_Id`, `order`, `type`, `delay`, `extent`, `motion`, `message`, `test_String`, `min`, `max`, `min_64`, `max_64`, `min_Dbl`, `max_Dbl`, `stat`, `display`, `amount`, `amount_64`, `hero_X_P_64`, `percent`, `spell_Id`, `wealth_Rating`, `treasure_Class`, `treasure_Type`, `p_Script`, `sound`, `destination_Type`, `weenie_Class_Id`, `stack_Size`, `palette`, `shade`, `try_To_Bond`, `obj_Cell_Id`, `origin_X`, `origin_Y`, `origin_Z`, `angles_W`, `angles_X`, `angles_Y`, `angles_Z`)
+VALUES (@parent_id,  0,  19 /* CastSpellInstant */, 0, 1, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 0, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL);
+
+INSERT INTO `weenie_properties_create_list` (`object_Id`, `destination_Type`, `weenie_Class_Id`, `stack_Size`, `palette`, `shade`, `try_To_Bond`)
+VALUES (87150, 9, 87151,  0, 0, 0.15, False) /* Create Blue Face Paint Recipe (87151) for ContainTreasure */
+     , (87150, 9,     0,  0, 0, 0.85, False) /* Create nothing for ContainTreasure */;
```
Comment (ziang4891) [inline Database/Patches/2006-10-DanceOfTheDead/9 WeenieDefaults/Creature/Unsorted/87150 Storage Crate.sql:178]: Being that this mob is NoCorpse, a ContainTreasure is never going to work.  You will need to have a separate event generator that this crate activates on death and then deactivates on generation.

**Review by ziang4891 — CHANGES_REQUESTED** [review]

File: Database/Patches/2006-10-DanceOfTheDead/9 WeenieDefaults/Creature/Fiun/32595 Fiun Layeel.sql [hunk 2]
```diff
@@ -50,6 +50,84 @@ VALUES (32595,   1, 135, 0, 0) /* Strength */
      , (32595,   6, 130, 0, 0) /* Self */;
 
 INSERT INTO `weenie_properties_attribute_2nd` (`object_Id`, `type`, `init_Level`, `level_From_C_P`, `c_P_Spent`, `current_Level`)
-VALUES (32595,   1,     0, 0, 0, 52) /* MaxHealth */
-     , (32595,   3,     0, 0, 0, 103) /* MaxStamina */
-     , (32595,   5,     0, 0, 0, 130) /* MaxMana */;
+VALUES (32595,   1,     0, 0, 0, 62) /* MaxHealth */
+     , (32595,   3,     0, 0, 0, 113) /* MaxStamina */
+     , (32595,   5,     0, 0, 0, 140) /* MaxMana */;
+
```
[the remark this answers is not in the document]
Comment (sLackeR-hub) [reply Database/Patches/2006-10-DanceOfTheDead/9 WeenieDefaults/Creature/Fiun/32595 Fiun Layeel.sql:56]: Fixed

File: Database/Patches/2006-10-DanceOfTheDead/9 WeenieDefaults/Portal/32811 Bookcase.sql [hunk 3]
```diff
@@ -0,0 +1,36 @@
+DELETE FROM `weenie` WHERE `class_Id` = 32811;
+
+INSERT INTO `weenie` (`class_Id`, `class_Name`, `type`, `last_Modified`)
+VALUES (32811, 'ace32811-bookcase', 7, '2019-02-04 06:52:23') /* Portal */;
+
+INSERT INTO `weenie_properties_int` (`object_Id`, `type`, `value`)
+VALUES (32811,   1,      65536) /* ItemType - Portal */
+     , (32811,  16,         32) /* ItemUseable - Remote */
+     , (32811,  53,        101) /* PlacementPosition - Resting */
```
[the remark this answers is not in the document]
Comment (sLackeR-hub) [reply Database/Patches/2006-10-DanceOfTheDead/9 WeenieDefaults/Portal/32811 Bookcase.sql:9]: Removed

File: Database/Patches/2006-10-DanceOfTheDead/9 WeenieDefaults/Portal/32811 Bookcase.sql [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+     , (32811, 111,         49) /* PortalBitmask - Unrestricted, NoSummon, NoRecall */;
+
+INSERT INTO `weenie_properties_bool` (`object_Id`, `type`, `value`)
+VALUES (32811,   1, True ) /* Stuck */
+     , (32811,  13, True ) /* Ethereal */
+     , (32811,  14, True ) /* GravityStatus */
+     , (32811,  15, True ) /* LightsStatus */
+     , (32811,  19, True ) /* Attackable */
+     , (32811,  88, True ) /* PortalShowDestination */;
+
```
[the remark this answers is not in the document]
Comment (sLackeR-hub) [reply Database/Patches/2006-10-DanceOfTheDead/9 WeenieDefaults/Portal/32811 Bookcase.sql:21]: Fixed.

File: Database/Patches/2006-10-DanceOfTheDead/9 WeenieDefaults/Creature/Fiun/32595 Fiun Layeel.sql (hunk 2 above)
[the remark this answers is not in the document]
Comment (ziang4891) [reply Database/Patches/2006-10-DanceOfTheDead/9 WeenieDefaults/Creature/Fiun/32595 Fiun Layeel.sql:56]: Still not fixed

**Review by ziang4891 — APPROVED** [review]

**Review by ziang4891 — APPROVED** [review]

**Pull request merged**